Firestore/core/src/firebase/firestore/remote/grpc_connection.cc

```diff
@@ -207,9 +208,10 @@ std::shared_ptr<grpc::Channel> GrpcConnection::CreateChannel() const {
   }
 
   // For tests only
+  auto fs = Filesystem::Default();
```

Comment:
I've added the asterisk.

I haven't removed the variable because:
  * In all cases inlining `Filesystem::Default()` in place of `fs` causes a line wrap; and
  * In most cases I expect that eventually those usages will migrate to a member variable anyway